src/accidental.js

```diff
@@ -465,36 +465,42 @@ export class Accidental extends Modifier {
 
   // Render accidental onto canvas.
   draw() {
-    if (!this.context) {
+    const {
+      context,
+      type, position, note, index, cautionary,
+      x_shift, y_shift, width,
+      accidental, parenLeft, parenRight,
+      render_options: { font_scale },
+    } = this;
```

Comment:
I find it helpful to logically group properties when destructuring, particularly with a lot of properties. But it's not necessary.